collectors/logsmanagement.plugin/plugin_logsmanagement_web_log.h

```diff
@@ -0,0 +1,135 @@
+/** @file plugins_logsmanagement_web_log.h
+ *  @brief Incudes the structure and function definitions to use web log charts.
+ *
+ *  @author Dimitris Pantazis
+ */
+
+#ifndef PLUGIN_LOGSMANAGEMENT_WEB_LOG_H_
+#define PLUGIN_LOGSMANAGEMENT_WEB_LOG_H_
+
+#include "../../daemon/common.h"
+#include "../../logsmanagement/file_info.h"
+#include "../../logsmanagement/circular_buffer.h"
+
+typedef struct Chart_data_web_log chart_data_web_log_t;
+
+#include "plugin_logsmanagement.h"
+
+/* NETDATA_CHART_PRIO for Chart_data */
```

Comment:
Please, move there `PRIO` like I suggested above.

Reply:
Done, please see https://github.com/netdata/netdata/pull/13291/commits/60ad8f86f1116a8416abc96878c08c65db18bbc4 as suggested in my other comment.